prov/efa/src/efa_hmem.c

```diff
@@ -125,20 +188,21 @@ static int efa_hmem_support_status_update_neuron(struct efa_hmem_support_status
 	/*
 	 * neuron_alloc will fail if application did not call nrt_init,
 	 * which is ok if it's not running neuron workloads. libfabric
-	 * will move on and leave neuron_status->initialized as false.
+	 * will move on and leave neuron_info->initialized as false.
 	 */
 	if (!ptr) {
 		EFA_INFO(FI_LOG_DOMAIN,
 			"Cannot allocate Neuron buffer. \n");
 		return 0;
 	}
 
-	neuron_status->initialized = true;
+	neuron_info->initialized = true;
 
 	ibv_mr = ibv_reg_mr(g_device_list[0].ibv_pd, ptr, len, ibv_access);
 	if (!ibv_mr) {
 		EFA_WARN(FI_LOG_DOMAIN,
-			 "Failed to register Neuron buffer with the EFA device, FI_HMEM transfers that require peer to peer support will fail.\n");
+		         "Failed to register Neuron buffer with the EFA device, "
+		         "FI_HMEM transfers that require peer to peer support will fail.\n");
```

Comment:
I think I will add a commit in the future to remove this registration because we do not expect to get neuron point on a non-p2p system.